src/test/java/com/google/api/codegen/testdata/java_settings_library.baseline

```diff
@@ -717,9 +717,6 @@ public class LibraryServiceSettings extends ServiceApiSettings {
           addCommentsSettings,
           getBookFromArchiveSettings,
           updateBookIndexSettings,
-          streamShelvesSettings,
```

Comment:
Those settings are streaming settings which should not be a part of unaryMethodSettingsBuilders array.